### PR TITLE
Fix clickhouse-keeper-client symlink

### PR DESCRIPTION
**Pull request by deric**
Since #50964  got merged there's 

```
$ clickhouse-keeper client 
```
command which makes the `clickhouse-keeper-client` symlink obsolete.

Moreover `clickhouse-keeper` package contains following files:

```
$ dpkg-query -L clickhouse-keeper
/etc
/etc/clickhouse-keeper
/etc/clickhouse-keeper/keeper_config.xml
/lib
/lib/systemd
/lib/systemd/system
/lib/systemd/system/clickhouse-keeper.service
/usr
/usr/bin
/usr/bin/clickhouse-keeper
/usr/share
/usr/share/doc
/usr/share/doc/clickhouse-keeper
/usr/share/doc/clickhouse-keeper/AUTHORS
/usr/share/doc/clickhouse-keeper/CHANGELOG.md
/usr/share/doc/clickhouse-keeper/LICENSE
/usr/share/doc/clickhouse-keeper/README.md
/usr/bin/clickhouse-keeper-client
/usr/bin/clickhouse-keeper-converter
```
which means that the symlink would be broken anyway (unless some other package provides `/usr/bin/clickhouse` binary)
```
$ ll /usr/bin/clickhouse*
-rwxr-xr-x 1 root root 25580984 Sep  4 18:09 /usr/bin/clickhouse-keeper*
lrwxrwxrwx 1 root root       10 Sep  4 18:31 /usr/bin/clickhouse-keeper-client -> clickhouse
lrwxrwxrwx 1 root root       10 Sep  4 18:31 /usr/bin/clickhouse-keeper-converter -> clickhouse
```
Same applies for `clickhouse-keeper-converter` symlink.

### Changelog category (leave one):
- Build/Testing/Packaging Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Remove redundant `clickhouse-keeper-client` symlink

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by deric** [comment]
I'm afraid simple symlink won't do, `clickhouse-keeper-client` would need to be defined e.g. as:
```
#!/bin/bash
clickhouse-keeper client "$@"
```

**Comment by Felixoid** [comment]
Here's what I've done and it works

```
docker run -it --rm ubuntu:22.04 bash
root@6cf7f71b05f1:/# apt update && apt install curl
root@6cf7f71b05f1:/# curl -O https://packages.clickhouse.com/deb/pool/main/c/clickhouse/clickhouse-keeper_23.8.2.7_amd64.deb
root@6cf7f71b05f1:/# dpkg -i clickhouse-keeper_23.8.2.7_amd64.deb
root@6cf7f71b05f1:/# clickhouse-keeper client
Coordination::Exception: All connection tries failed while connecting to ZooKeeper. nodes: [::1]:9181
Poco::Exception. Code: 1000, e.code() = 111, Connection refused (version 23.8.2.7 (official build)), [::1]:9181
Poco::Exception. Code: 1000, e.code() = 111, Connection refused (version 23.8.2.7 (official build)), [::1]:9181
Poco::Exception. Code: 1000, e.code() = 111, Connection refused (version 23.8.2.7 (official build)), [::1]:9181

root@6cf7f71b05f1:/# clickhouse-keeper-client
bash: clickhouse-keeper-client: command not found
root@6cf7f71b05f1:/# ln -sf /usr/bin/clickhouse-keeper usr/bin/clickhouse-keeper-client
root@6cf7f71b05f1:/# clickhouse-keeper-client
Coordination::Exception: All connection tries failed while connecting to ZooKeeper. nodes: [::1]:9181
Poco::Exception. Code: 1000, e.code() = 111, Connection refused (version 23.8.2.7 (official build)), [::1]:9181
Poco::Exception. Code: 1000, e.code() = 111, Connection refused (version 23.8.2.7 (official build)), [::1]:9181
Poco::Exception. Code: 1000, e.code() = 111, Connection refused (version 23.8.2.7 (official build)), [::1]:9181

root@6cf7f71b05f1:/# ln -sf /usr/bin/clickhouse-keeper usr/bin/clickhouse-keeper-converter
root@6cf7f71b05f1:/# clickhouse-keeper-converter
Processing configuration file 'keeper_config.xml'.
Code: 107. DB::Exception: Configuration file keeper_config.xml doesn't exist and there is no embedded config. (FILE_DOESNT_EXIST), Stack trace (when copying this message, always include the lines below):

0. DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x0000000000e1185b in /usr/bin/clickhouse-keeper
1. DB::Exception::Exception<String const&>(int, FormatStringHelperImpl<std::type_identity<String const&>::type>, String const&) @ 0x000000000078f9cd in /usr/bin/clickhouse-keeper
2. DB::ConfigProcessor::processConfig(bool*, zkutil::ZooKeeperNodeCache*, std::shared_ptr<Poco::Event> const&) @ 0x0000000000db77b7 in /usr/bin/clickhouse-keeper
3. DB::ConfigProcessor::loadConfig(bool) @ 0x0000000000db8b34 in /usr/bin/clickhouse-keeper
4. BaseDaemon::initialize(Poco::Util::Application&) @ 0x0000000000a28231 in /usr/bin/clickhouse-keeper
5. DB::Keeper::initialize(Poco::Util::Application&) @ 0x0000000000b4d2c3 in /usr/bin/clickhouse-keeper
6. Poco::Util::Application::run() @ 0x0000000000fb6dfa in /usr/bin/clickhouse-keeper
7. DB::Keeper::run() @ 0x0000000000b4d11e in /usr/bin/clickhouse-keeper
8. Poco::Util::ServerApplication::run(int, char**) @ 0x0000000000fcdf19 in /usr/bin/clickhouse-keeper
9. mainEntryClickHouseKeeper(int, char**) @ 0x0000000000b4c098 in /usr/bin/clickhouse-keeper
10. main @ 0x0000000000b5ae79 in /usr/bin/clickhouse-keeper
 (version 23.8.2.7 (official build))
```

So, symlinks should be fixed, and not removed. Good catch, thanks!

File: packages/clickhouse-keeper.yaml [hunk 1]
```diff
@@ -32,9 +32,6 @@ contents:
   dst: /usr/bin/clickhouse-keeper
 - src: clickhouse-keeper.service
   dst: /lib/systemd/system/clickhouse-keeper.service
-- src: clickhouse
-  dst: /usr/bin/clickhouse-keeper-client
-  type: symlink
 - src: clickhouse
```
Comment (Felixoid) [inline packages/clickhouse-keeper.yaml:35]:
```suggestion
- src: clickhouse-keeper
```

**Review by Felixoid — CHANGES_REQUESTED** [review]
Please, bring the symlink back with the correct src `clickhouse-keeper`

File: packages/clickhouse-keeper.yaml (hunk 1 above)
Comment (deric) [reply packages/clickhouse-keeper.yaml]: Fixed

**Comment by deric** [comment]
@Felixoid You're right, that's definitely a better solution.

**Review by Felixoid — APPROVED** [review]

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit 7a12488db5ee5d5289dc004904dd7567e627ad0a with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/54587/7a12488db5ee5d5289dc004904dd7567e627ad0a/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>✅ success</td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>⏳ pending</td></tr>
<tbody>
</table>

**Pull request merged**